src/com/chessclock/android/ChessClock.java

```diff
@@ -747,13 +766,9 @@ private void setUpGame(boolean resetClocks) {
             return;
         }
 	    
-		delay = prefs.getString("prefDelay","None");
-		if ( delay.equals("")) {
-			delay = "None";
-			Editor e = prefs.edit();
-			e.putString("prefDelay", "None");
-			e.commit();
-		}
```

Comment:
Please correct me if I'm wrong, but I don't think `delay` can ever be an empty string, so this check seemed unnecessary.